openmdao/core/test/test_src_indices.py

```diff
@@ -122,14 +122,13 @@ def test_src_indices_connect_error(self):
         root.connect('P.x', 'G.A.x', src_indices=[0])
         root.connect('P.x', 'C.x', src_indices=[2,])
 
-        expected_error_message = py3fix("Size 1 of the indexed sub-part of "
-                                        "source 'P.x' must match the size "
-                                        "'2' of the target 'G.A.x'")
         prob = Problem(root)
         with self.assertRaises(ConnectError) as cm:
             prob.setup(check=False)
 
-        self.assertEqual(str(cm.exception), expected_error_message)
+        expected = py3fix("Size 1 of the indexed sub-part of source 'P.x' "
+                          "must be the same as size '2' of target 'G.A.x'")
```

Comment:
I don't think we should put quotes around a size, or really anything other than a string.